custom/icds_reports/dashboard_utils.py

```diff
@@ -25,4 +29,30 @@ def get_dashboard_template_context(domain, couch_user):
     if couch_user.is_web_user():
         context['is_web_user'] = True
 
+    context['nav_metadata'] = _get_nav_metadatada()
     return context
+
+
+def _get_nav_metadatada():
```

Comment:
Basic idea here is to start standardizing / centralizing this logic rather than duplicating it across random javascript files in different places. Just started with the one report I was working on.